rocky/katalogus/client.py

```diff
@@ -18,24 +18,41 @@
 logger = getLogger(__name__)
 
 
-class Plugin(BaseModel):
+class Boefje(BaseModel):
     id: str
-    type: str
+    repository_id: Optional[str] = None
     name: str
-    description: str
-    repository_id: str
+    description: Optional[str] = None
+    enabled: bool
+    type: str
     scan_level: SCAN_LEVEL
     consumes: Set[Type[OOI]]
     produces: Set[Type[OOI]]
-    enabled: bool = True
 
     def dict(self, *args, **kwargs):
         """Pydantic does not stringify the OOI classes, but then templates can't render them"""
-        plugin_dict = super().dict(*args, **kwargs)
-        plugin_dict["consumes"] = {ooi_class.get_ooi_type() for ooi_class in plugin_dict["consumes"]}
-        plugin_dict["produces"] = {ooi_class.get_ooi_type() for ooi_class in plugin_dict["produces"]}
+        boefje_dict = super().dict(*args, **kwargs)
+        boefje_dict["consumes"] = {ooi_class.get_ooi_type() for ooi_class in boefje_dict["consumes"]}
+        boefje_dict["produces"] = {ooi_class.get_ooi_type() for ooi_class in boefje_dict["produces"]}
+
+        return boefje_dict
 
-        return plugin_dict
+
+class Normalizer(BaseModel):
```

Comment:
Why are we not re-using the `Plugin` model? The Katalogus itself also defines this generic plugin model and now we copy over all fields while the `type` field already specifies the kind of plugin.

Reply:
I have refactored this part

Reply:
I see the difference now indeed, perhaps this requires an API update for the Katalogus as well to clean this up but I think this is ok for now